integration/query_projection_compat_test.go

```diff
@@ -24,17 +24,68 @@ func TestQueryProjectionCompat(t *testing.T) {
 	t.Parallel()
 
 	testCases := map[string]queryCompatTestCase{
-		"FindProjectionInclusions": {
-			filter:         bson.D{{"_id", "document-composite"}},
-			projection:     bson.D{{"foo", int32(1)}, {"42", true}},
-			skipForTigris:  "Tigris does not support field names started from numbers (`42`)",
-			resultPushdown: true,
-		},
-		"FindProjectionExclusions": {
-			filter:         bson.D{{"_id", "document-composite"}},
-			projection:     bson.D{{"foo", int32(0)}, {"array", false}},
-			skipForTigris:  "Tigris does not support language keyword 'array' as field name",
-			resultPushdown: true,
+		"Include1Field": {
+			filter:     bson.D{},
+			projection: bson.D{{"v", int32(1)}},
+		},
+		"Exclude1Field": {
+			filter:     bson.D{},
+			projection: bson.D{{"v", int32(0)}},
+		},
+		"Include2Fields": {
+			filter:     bson.D{},
+			projection: bson.D{{"foo", int32(1)}, {"bar", true}},
+		},
+		"Exclude2Fields": {
+			filter:     bson.D{},
+			projection: bson.D{{"foo", int32(0)}, {"bar", false}},
+		},
+		"IncludeID": {
+			filter:     bson.D{},
+			projection: bson.D{{"_id", true}},
+		},
+		"ExcludeID": {
+			filter:     bson.D{},
+			projection: bson.D{{"_id", false}},
+		},
+		"IncludeFieldExcludeID": {
+			filter:     bson.D{},
+			projection: bson.D{{"_id", false}, {"array", true}},
+		},
+		"ExcludeFieldIncludeID": {
+			filter:     bson.D{},
+			projection: bson.D{{"_id", true}, {"array", false}},
+		},
```

Comment:
Could you add test for 
`projection: bson.D{{"_id", true}, {"v", true}},` I'm curious because of https://github.com/FerretDB/FerretDB/pull/2386/files#diff-7bba61640a80f2f9020b4e4049d8a46e197e35d987264d20e1ed1cfe41121f9dR108.
Also could you add `projection: bson.D{{"_id", false}, {"v", false}},`? I don't know what happens if everything is excluded.